lib/Dancer2/Plugin/Passphrase/Core.pm

```diff
@@ -72,8 +72,11 @@ sub _merge_options {
     if ( $algorithm eq 'Bcrypt' ) {
         $settings->{'scheme'} = 'CRYPT';
         $settings->{'type'}   = '2a';
-        $settings->{'cost'} =
-          defined $settings->{'cost'} ? $settings->{'cost'} : 4;
+
+        unless ( defined $settings->{'cost'} ) {
+            $settings->{'cost'} = $self->{'Bcrypt'}{'cost'} || 4;
```

Comment:
Can we just write:

```perl
$settings->{'cost'} //= $self->{'Bcrypt'}{'cost'} || 4;
```

Also, are the other configurations made available here too? If they aren't, perhaps we should.

Reply:
What version of Perl did //= come to be in? I know we try to support everything going back to 5.8, but I thought //= came around after that. Do we care if plugins we maintain don't support 5.8 even though Dancer2 does?

Reply:
(that last comment could just be my cold meds talking)

Reply:
`//` came with Perl 5.10.

As for should we support 5.8 for plugins, my personal opinion is that 5.10 came out *10* years ago... if that's not long enough, ye gods, then how long is long enough?

Reply:
I am not disagreeing. Just pointing out that it is something we have done up to this point. Changing that isn't a discussion for this thread though :)

Reply:
I'm inclined to believe that `//=` is better here.